config/locales/zh-TW.yml

```diff
@@ -59,6 +29,7 @@ zh-TW:
       help:
         redirect_uri: '每個網址必須只有一行'
         native_redirect_uri: '使用 %{native_redirect_uri} 作本地端測試'
+        scopes: 'Separate scopes with spaces. Leave blank to use the default scopes.'
```

Comment:
Let's leave the translations we don't have as blank. I believe that's encoded as `~`.